hooks/hiero_post_version_creation.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from tank import Hook
+
+class HieroPostVersionCreation(Hook):
```

Comment:
FYI, the preferred way to do inheritance in hooks can be found here:
https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/master/hooks/custom_actions.py#L16-L18
This is not as important here since this is meant to be the base class, but when chaining classes in the config with inheritance chains, this is the magic that makes everything derive from the right base.